tdrs-backend/tdpservice/parsers/util.py

```diff
@@ -281,3 +283,220 @@ def parse_and_validate(self, line, generate_error):
             records.append(r)
 
         return records
+
+def get_schema_options(program, section, query=None, model=None, model_name=None):
+    """Centralized function to return the appropriate schema for a given program, section, and query.
+
+    TODO: need to rework this docstring as it is outdated hence the weird ';;' for some of them.
+
+    @param program: the abbreviated program type (.e.g, 'TAN')
+    @param section: the section of the file (.e.g, 'A');; or ACTIVE_CASE_DATA
+    @param query: the query for section_names (.e.g, 'section', 'models', etc.)
+    @return: the appropriate references (e.g., ACTIVE_CASE_DATA or {t1,t2,t3}) ;; returning 'A'
+    """
+    schema_options = {
+        'TAN': {
+            'A': {
+                'section': DataFile.Section.ACTIVE_CASE_DATA,
+                'models': {
+                    'T1': schema_defs.tanf.t1,
+                    'T2': schema_defs.tanf.t2,
+                    'T3': schema_defs.tanf.t3,
+                }
+            },
+            'C': {
+                'section': DataFile.Section.CLOSED_CASE_DATA,
+                'models': {
+                    # 'T4': schema_defs.tanf.t4,
+                    # 'T5': schema_defs.tanf.t5,
+                }
+            },
+            'G': {
+                'section': DataFile.Section.AGGREGATE_DATA,
+                'models': {
+                    # 'T6': schema_defs.tanf.t6,
+                }
+            },
+            'S': {
+                'section': DataFile.Section.STRATUM_DATA,
+                'models': {
+                    # 'T7': schema_defs.tanf.t7,
+                }
+            }
+        },
+        'SSP': {
+            'A': {
+                'section': DataFile.Section.SSP_ACTIVE_CASE_DATA,
+                'models': {
+                    'M1': schema_defs.ssp.m1,
+                    'M2': schema_defs.ssp.m2,
+                    'M3': schema_defs.ssp.m3,
+                }
+            },
+            'C': {
+                'section': DataFile.Section.SSP_CLOSED_CASE_DATA,
+                'models': {
+                    # 'S4': schema_defs.ssp.m4,
+                    # 'S5': schema_defs.ssp.m5,
+                }
+            },
+            'G': {
+                'section': DataFile.Section.SSP_AGGREGATE_DATA,
+                'models': {
+                    # 'S6': schema_defs.ssp.m6,
+                }
+            },
+            'S': {
+                'section': DataFile.Section.SSP_STRATUM_DATA,
+                'models': {
+                    # 'S7': schema_defs.ssp.m7,
+                }
+            }
+        },
+        # TODO: tribal tanf
+    }
+
+    if query == "text":
```

Comment:
I alway lean to have a file with all constants defined and imported. This makes the code more maintainable in the future.